react-javascript/src/App.jsx

```diff
@@ -1,21 +1,18 @@
-import * as React from 'react';
 import './App.css';
```

Comment:
Isn't it still best practice to include `import React from 'react';` at the top of a `jsx` even if it isn't needed anymore by the default react builder. Just for compatibility for other bundlers and builders.

Reply:
VSCode suggested to remove it and compiler wasn't complaining but true, will reinstate.

Reply:
Let's leave it out then. It's probably just an old convention.